src/hooks/useConnection.js

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useReducer, useRef } from "react";
+import Peer from "simple-peer";
+
+import { audioRefsAction, peersAction } from "../reducer/actions";
+import { audioRefsReducer, peersReducer } from "../reducer";
+import { CHANNEL } from "../config/constants";
+import useSocket from "./useSocket";
+
+export default function useConnection(channelId) {
+  const [audioRefs, audioRefsDispatch] = useReducer(audioRefsReducer, []);
+  const [peers, peersDispatch] = useReducer(peersReducer, []);
+  const socket = useSocket();
+  const myAudio = useRef();
+
+  useEffect(() => {
+    try {
+      const connectRTC = async () => {
+        const stream = await navigator.mediaDevices.getUserMedia({
+          video: true,
+          audio: true,
+        });
```

Comment:
저희는 audio장치만 필요할 것 같습니다!
```suggestion
        const stream = await navigator.mediaDevices.getUserMedia({
          audio: true,
        });
```

Reply:
맞아요! 잘봐주셨네요 비디오로 테스트를 거치고 수정을 안해줬습니다 👍  꼼꼼하십니다